### ISSUE TITLE
Replace `Github` with `GitHub` in _projects/ems-triage-tracker.md

### ISSUE THREAD

**Issue by daras-cu** [hackforla/website#7413]
### Overview
We need to replace instances of `Github` and `github` with `GitHub` so that we display the company name correctly, and test the changes to ensure the behavior and appearance of any and all related webpages are unchanged.    

### Details
The instance of "Github" that you are changing seems to relate to the creation of this button on the https://www.hackforla.org/communities-of-practice page 
<img src="https://github.com/user-attachments/assets/11006ff7-c45c-4763-9a34-4ebac615cec3" width=200px> 
that is created by https://github.com/hackforla/website/blob/gh-pages/pages/communities-of-practice.html.

### Action Items
- [x] Open `_projects/ems-triage-tracker.md` in your IDE
- [x] Change
```
  - name: Github
```
to
```
  - name: GitHub
```
- [x] Find all the files that are connected to this change.  (e.g., if its a yml or md file, what files or scripts use the data stored in that file).  And document briefly how they work together.
- [x] Define where you expect the change to show up on the website (e.g., the footer of the website).  Can be on multiple pages
- [x] Write a test procedure, in a comment, and then use it to test the change.  Include a link to this comment in your PR.
- [x] Add/commit changes then create a PR as usual.  Document the test procedure in the Pull Request.  

### Resources
- This issue resulted from #6161

### Merge Team
- [x] After closing this issue, check off the dependency in #6161.  If all dependencies have been completed, close #6161

**Comment by Priyanshu-2005** [comment]
Hey @daras-cu  I would like to work on this issue kindly assign me this issue.

**Comment by daras-cu** [comment]
Hi @Priyanshu-2005, have you done onboarding for the website team and worked on issues before? Take a look at the [Getting Started Page](https://www.hackforla.org/getting-started) and the [Wiki](https://github.com/hackforla/website/wiki) for info about onboarding and working on issues. This issue is waiting on prioritization and not ready to be assigned yet.

**Comment by HackforLABot** [comment]
Hi @FamousHero, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)
